trikot-kword/kword-remote-update/src/androidMain/kotlin/com/mirego/trikot/kword/remote/update/KwordRemoteUpdate.kt

```diff
@@ -0,0 +1,27 @@
+package com.mirego.trikot.kword.remote.update
+
+import android.content.Context
+import com.mirego.trikot.kword.I18N
+import okio.FileSystem
+
+actual object KwordRemoteUpdate {
```

Comment:
Having this as an object means we cannot load translations from multiple remote locations at the same time.

In a setup where we'd have multiple translation sets (aka multiple modules) I feel I'd be better to have this as a class so we load from several sources at the same time.